base/server/src/main/java/org/dogtagpki/server/rest/JobService.java

```diff
@@ -102,6 +148,16 @@ public Response getJob(String id) throws EBaseException {
             throw new ResourceNotFoundException("Job " + id + " not found");
         }
 
+        Principal principal = servletRequest.getUserPrincipal();
+        logger.info("JobService: - principal: " + principal);
+
+        boolean isAdmin = isAdmin(principal);
+        boolean isOwner = isOwner(principal, jobConfig);
+
+        if (!isAdmin && !isOwner) {
+            throw new ForbiddenException();
```

Comment:
Can we add a message in explaining why this exception was thrown?

Reply:
The default HTTP error message seems to be self explanatory:

```
$ pki ca-job-show unpublishExpiredCerts
PKIException: Forbidden
```

Do you have a suggestion? In general I'm not sure if we should provide a detailed reason why a user is unauthorized to execute an operation. In the future the job owner could be a list of groups. There's a risk it might expose a weakness in the system if we return that list to the user.

Reply:
Good point, better to leave it as it is :-)